UnityProject/Assets/Scripts/Health/Living/HumanHealth/PlayerHealth.cs

```diff
@@ -14,9 +15,31 @@ public class PlayerHealth : LivingHealthBehaviour
 	[SerializeField]
 	private MetabolismSystem metabolism;
```

Comment:
Missed this one

Reply:
Ahah!  Never saw that one since it's not of my own.  Solved it.